src/Kotlin/DocumentationBuilder.kt

```diff
@@ -128,7 +175,11 @@ class DocumentationBuilder(val session: ResolveSession, val options: Documentati
         for ((packageName, declarations) in descriptors) {
             println("  package $packageName: ${declarations.count()} nodes")
```

Comment:
Shouldn't println here be fixed to adhere to different count of nodes generated below?